lib/widgets/emoji_animation.dart

```diff
@@ -0,0 +1,58 @@
+import 'dart:async';
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class EmojiAnimation extends StatefulWidget {
+  List<String> emoji = [
+    'assets/emoji1.png',
+    'assets/emoji2.png',
+    'assets/emoji3.png',
+    'assets/emoji4.png',
+    'assets/emoji5.png',
+    'assets/emoji6.png',
+    'assets/emoji7.png',
+  ];
+
+  EmojiAnimation({Key? key}) : super(key: key);
+
+  @override
+  State<EmojiAnimation> createState() => _EmojiAnimationState();
+}
+
+class _EmojiAnimationState extends State<EmojiAnimation> {
+  int _index = 0;
+  late Timer _timer;
+
+  @override
+  void initState() {
+    _timer = Timer.periodic(new Duration(seconds: 5), (Timer timer) {
+      setState(() {
+        _index = (_index + 1) % widget.emoji.length;
```

Comment:
why you are randomizing initial emoji here? I think initial emoji should be taken on some data context, like time spent or smth like that. If not - I would just leave first emoji here without combining it like that